### PR TITLE
Fix storage

### PR DESCRIPTION
**Pull request by jievince**

### CODE REVIEW

File: examples/StorageClientExample.py [hunk 1]
```diff
@@ -23,6 +24,7 @@
 
 def scan_edge(space, return_cols, all_cols):
     scan_edge_response_iter = storage_client.scan_edge(space, return_cols, all_cols, 100, 0, sys.maxsize)
+    print('############## scanned edge data ##############')
```
Comment (darionyaphet) [inline examples/StorageClientExample.py:27]: Do you mind using `LOG` to replace `print` ?

File: nebula/ngStorage/StorageClient.py [hunk 2]
```diff
@@ -200,15 +204,22 @@ def connect(self, address):
...
         if address not in self._clients.keys():
-            client = self.do_connect(address)
-            self._clients[address] = client
-            return client
+            while retry > 0:
+                client = self.do_connect(address)
+                if client != None:
+                    self._clients[address] = client
+                    return client
+                retry -= 1
```
Comment (laura-ding) [inline nebula/ngStorage/StorageClient.py:214]: If retry failed, here without return object. add `return None`

File: nebula/ngStorage/StorageClient.py [hunk 3]
```diff
@@ -200,15 +204,22 @@ def connect(self, address):
...
         else:
             return self._clients[address]
 
     def disconnect(self, address):
```
Comment (laura-ding) [inline nebula/ngStorage/StorageClient.py:218]: if the client is `None`, meaning the client is not added into the `_clients`, of course not need to call disconnect. Pay attention to the logic.

File: examples/StorageClientExample.py (hunk 1 above)
Comment (jievince) [reply examples/StorageClientExample.py:27]: Good idea!

File: nebula/ngStorage/StorageClient.py (hunk 2 above)
Comment (jievince) [reply nebula/ngStorage/StorageClient.py:215]: OK

**Review by darionyaphet — APPROVED** [review]

**Pull request merged**